src/olympia/blocklist/forms.py

```diff
@@ -77,6 +77,19 @@ class BlocksWidget(forms.widgets.SelectMultiple):
     def optgroups(self, name, value, attrs=None):
         return []
 
+    def get_verb(self, action):
+        """Return the verb to use when displaying a given version, depending
+        on the action.
+
+        Re-uses BlocklistSubmission action display strings, but in a shorter
+        form, adapted for displaying next to a single version."""
+        return {
+            BlocklistSubmission.ACTION_ADDCHANGE: 'Block',
+            BlocklistSubmission.ACTION_DELETE: 'Unblock',
+            BlocklistSubmission.ACTION_HARDEN: 'Harden',
+            BlocklistSubmission.ACTION_SOFTEN: 'Soften',
```

Comment:
certainly not a hill I'd die on, but really these terms should all be defined in one place, imo